x/epoching/types/params.go

```diff
@@ -8,16 +8,32 @@ const (
 	DefaultEpochInterval uint64 = 10
 )
 
+const (
+	DefaultMinAmount uint64 = 1
+)
+
+var DefaultEnqueueGasFees = EnqueueGasFees{
+	Delegate:                  500,
+	Undelegate:                400,
+	BeginRedelegate:           600,
+	CancelUnbondingDelegation: 300,
+	EditValidator:             200,
+	StakingUpdateParams:       100,
+	CreateValidator:           800,
```

Comment:
The gas value seems odd; even if you don’t find the exact worst case right away, please at least reset it to a rough level that reflects realistic conditions